### ISSUE TITLE
Coming again to the CRLF question

### ISSUE THREAD

**Issue by badal** [rubocop/rubocop#250]
Just some remarks.

TL;DR : I think that EOL conventions are OS dependent, and do not belong to ruby.

When on Windows, text files lines end by CRLF. When you open a file in binary mode ("rb"), CRLF is read  as "\r\n" ; when you open it in normal mode ("r" or "w"), ruby converts CRLF in "\n" when reading, and "\n" in CRLF when writing. Source files are text files. 

This (in my opinion !) has two corollaries :
1. On Parser side. If Parser read source files (either by read or readlines) in normal mode, it should not see \r at all when running on Windows.Why an empty line (= CRLF) is not seen as empty (= \n) by Parser ?
2. On Rubocop side. Line endings are not a question of ruby style, but are OS dependent (\n on *nix, \r\n on Windows, \r on old Macs). The corresponding Cop should be dropped, IMHO (or converted to "lines should end by the correct OS end of line").

Rubocop is a wonderful tool ! Thanks a lot for providing it. I have been developing ruby projects on Windows for years, I am very happy with Rubocop, and I'd like it to become a must_use for ruby-on-Windows developers. General tools, like IDEs (RubyMine in my case) or Git (msgit) provide files with CRLF endings. Working on Windows should not be against good ruby style...

What do you think ?
